### PR TITLE
Add Info Component to Storybook

### PR DESCRIPTION
**Pull request by colefriedlaender**
Info Component

### CODE REVIEW

File: components/Info/Info.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  otherWords: string;
+  information: string;
+};
+
+function Info({ firstWord, otherWords, information }: InfoProps) {
+  return (
+    <div className={styles.container}>
+      <div className={styles.firstWord}>
+        {firstWord}
+        <span className={styles.secondWord}>{otherWords}</span>
```
Comment (fre-ben) [inline components/Info/Info.tsx]: Naming `firstWord` and `otherWords` seems a bit weird.
Maybe there's a better way to name these?

**Review by fre-ben — APPROVED** [review]
LGTM 💯

File: components/Info/Info.module.css [hunk 2]
```diff
@@ -0,0 +1,27 @@
+.container {
+  background-color: var(--text-button-color);
+  height: 107px;
```
Comment (phlgr) [inline components/Info/Info.module.css]: This `height` seems odd

File: components/Info/Info.module.css [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  border-radius: 25px;
+  display: grid;
+  grid-template-rows: 1fr 1fr;
+  padding-left: 20px;
+  padding-right: 20px;
+  box-shadow: 0 2px 5px;
+}
+.firstWord {
+  padding-top: 10px;
+  font-size: 15px;
```
Comment (phlgr) [inline components/Info/Info.module.css]: This should be responsive, use `em` instead

File: components/Info/Info.module.css [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  text-align: center;
+}
+.secondWord {
+  font-size: 15px;
```
Comment (phlgr) [inline components/Info/Info.module.css]: Same as the above

File: components/Info/Info.module.css [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+  text-align: center;
+  display: block;
+}
+.information {
+  font-size: 25px;
```
Comment (phlgr) [inline components/Info/Info.module.css]: Same as the above

File: components/Info/Info.tsx [hunk 6]
```diff
@@ -0,0 +1,20 @@
+import styles from "../Info/Info.module.css";
+export type InfoProps = {
+  firstWord: string;
+  otherWords: string;
+  information: string;
+};
+
+function Info({ firstWord, otherWords, information }: InfoProps) {
```
Comment (phlgr) [inline components/Info/Info.tsx]: `firstWord`, `otherWords` and `information` isn't perfect naming. The prop names should express what is inside them

**Review by phlgr — CHANGES_REQUESTED** [review]

File: components/Info/Info.tsx [hunk 7]
```diff
@@ -1,18 +1,18 @@
...
 export type InfoProps = {
   firstWord: string;
-  otherWords: string;
-  information: string;
+  restOfPhrase: string;
+  content: string;
 };
 
-function Info({ firstWord, otherWords, information }: InfoProps) {
+function Info({ firstWord, restOfPhrase, content }: InfoProps) {
```
Comment (phlgr) [inline components/Info/Info.tsx:8]: I still find these hard to understand. What is it you are trying to display here?
Maybe a deployment would help

**Pull request merged**